Source/Wikitivity.Agent/WikitivityAgentHelper.cs

```diff
@@ -5,43 +5,156 @@
 using kCura.Relativity.ImportAPI;
 using Newtonsoft.Json;
 using Relativity.API;
+using Relativity.Services.Objects;
+using Relativity.Services.Objects.DataContracts;
 using System;
 using System.Collections.Generic;
 using System.Data;
 using System.Net;
+using System.Threading.Tasks;
+using ObjectType = kCura.Relativity.Client.DTOs.ObjectType;
 
 namespace Wikitivity.Agent
 {
 	class WikitivityAgentHelper
 	{
 		public static string IAPIMessages = "";
-
-		public bool DeleteUploadedRequestsFromRelativity(IRSAPIClient proxy, int workspaceID, List<WikitivityUploadsAgent.SingleRequestObject> listOfCompletedUploads)
+		public static Guid WikitivityRDOGuid = new Guid("C6196733-E2A6-48F4-9443-37990972EBA3");
+		
+		public bool DeleteUploadedRequestsFromRelativityOM(IObjectManager proxy, int workspaceID, List<WikitivityUploadsAgent.SingleRequestObject> listOfCompletedUploads)

```

Comment:
Unless you are using both the RSAPI and OM code in the same RAP file. I would remove the suffix OM from the method names.